modules/openapi-generator/src/main/resources/typescript/types/ObjectParamAPI.mustache

```diff
@@ -7,7 +7,7 @@ import { Observable } from 'rxjs';
 
 {{#models}}
 {{#model}}
-import { {{{ classname }}} } from '../models/{{{ classFilename }}}{{extensionForDeno}}';
+import { {{{ classname }}} } from '{{{ importPath }}}{{extensionForDeno}}';
```

Comment:
There are still still some imports in the `types/*` templates, which explicitly reference the `models` directory, instead of taking it from the `importPath`.
I think we don't actually need those imports anymore. Can you please remove the lines of the form
```
import * as models from '../models/all{{extensionForDeno}}';
```
from the three files?

Reply:
hmm... It’s exactly as you said.
Remove lines from three files.